khard/formatter.py

```diff
@@ -18,12 +18,12 @@ class Formatter:
     FORMAT = "formatted_name"
 
     def __init__(self, display: str, preferred_email: List[str],
-                 preferred_phone: List[str], preferred_kind: str,
-                 show_nicknames: bool, parsable: bool) -> None:
+                 preferred_phone: List[str], show_nicknames: bool,
+                 parsable: bool) -> None:
         self._display = display
         self._preferred_email = preferred_email
         self._preferred_phone = preferred_phone
-        self._preferred_kind = preferred_kind
+        self._default_kind = "individual"
```

Comment:
If this is a constant you can move it out of `__init__`

Reply:
Valid point. Although this has already been reworked in #310 (see [`khard/carddav_object.py:60`](https://github.com/lucc/khard/pull/310/files#diff-a189618efcc4fcc11fb726bee1742df072e671aace70782e53e4b2b8c1074f70R60)), so it may make sense to merge the two PRs after all…